projects/packages/my-jetpack/_inc/components/modules-list/utils.ts

```diff
@@ -14,7 +14,7 @@ export const JETPACK_MODULES_NOT_FOR_MULTISITE = [ 'waf', 'wordads' ];
 export function getModuleStatus( $module: MyJetpackModule ) {
 	// If the module is not supported on multisite, we set the availability to false and provide a reason.
 	if ( getScriptData().site.is_multisite ) {
-		if ( ! JETPACK_MODULES_NOT_FOR_MULTISITE.includes( $module.module ) ) {
+		if ( JETPACK_MODULES_NOT_FOR_MULTISITE.includes( $module.module ) ) {
```

Comment:
This seems to have been working prior to [this commit](https://github.com/Automattic/jetpack/pull/44260/commits/8159e19353fe55d3077bffa44a871fbc2862b11d).